src/Core/Managed/netstandard1.3/Extensibility/Implementation/Platform/PlatformImplementation.cs

```diff
@@ -1,51 +1,28 @@
 ﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Platform
 {
     using System;
-    using System.Collections.Generic;
-    using Microsoft.ApplicationInsights.Extensibility.Implementation.External;
 
-    internal class PlatformImplementation : IPlatform
+    /// <summary>
+    /// The .Net standard 1.3 implementation of the <see cref="IPlatform"/> interface.
+    /// </summary>
+    internal sealed class PlatformImplementation : PlatformImplementationBase
     {
-        private IDebugOutput debugOutput = null;
-        
-        public IDictionary<string, object> GetApplicationSettings()
-        {
-            return null;
-        }
-
-        public string ReadConfigurationXml()
-        {
-            return null;
-        }
-
-        public ExceptionDetails GetExceptionDetails(Exception exception, ExceptionDetails parentExceptionDetails)
-        {
-            return ExceptionConverter.ConvertToExceptionDetails(exception, parentExceptionDetails);
-        }
-
         /// <summary>
-        /// Returns the platform specific Debugger writer to the VS output console.
+        /// The directory where the configuration file might be found.
         /// </summary>
-        public IDebugOutput GetDebugOutput()
+        protected override string ConfigurationXmlDirectory
         {
-            if (this.debugOutput == null)
+            get
             {
-                this.debugOutput = new TelemetryDebugWriter(); 
+                return AppContext.BaseDirectory;
             }
-            
-            return this.debugOutput;
-        }
-
-        public string GetEnvironmentVariable(string name)
-        {
-            return null;
         }
 
         /// <summary>
         /// Returns the machine name.
         /// </summary>
-        /// <returns>The machine name.</returns>
-        public string GetMachineName()
+        /// <returns>Null always (Environment.MachineName is not available on .Net Standard 1.3).</returns>
+        protected override string GetHostName()
         {
             return null;
```

Comment:
Yes, in Windows we can use COMPUTERNAME as a reasonable approximation. There was some discussion over that in the coreclr repo: https://github.com/dotnet/corefx/issues/4306 
As a result, I think .NetStandard 2.0 has it directly on the Environment object, just like desktop FX.